fairlearn/postprocessing/_randomized_classifiers.py

```diff
@@ -0,0 +1,461 @@
+"""Helper functions to construct and use randomized classifiers.
+
+TODO: this module will probably be substituted by the InterpolatedThresholder
```

Comment:
If we need to make adjustments to InterpolatedThresholder let's do it (unless it's too complicated)